pom.xml

```diff
@@ -143,9 +143,11 @@
         <plugin>
```

Comment:
Not needed for newer developer tools. You can bump plugin POm to the recent version and use the `hpi.compatibleSinceVersion` property

Reply:
Does it look better now?

Reply:
@oleg-nenashev ping

Reply:
No. You need a very recent Plugin POM. You still have a version which is few years old